cli/pcluster/configure/easyconfig.py

```diff
@@ -0,0 +1,424 @@
+# Copyright 2013-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the 'LICENSE.txt' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+# fmt: off
+from __future__ import absolute_import, print_function  # isort:skip
+
+
+import copy
+import errno
+import logging
+import os
+import stat
+import tempfile
+
+import configparser
+
+from pcluster import cfnconfig
+from pcluster.configure.networking import (
+    NetworkConfiguration,
+    PublicPrivateNetworkConfig,
+    automate_subnet_creation,
+    automate_vpc_with_subnet_creation,
+    ec2_conn,
+)
+from pcluster.configure.utils import get_regions, get_resource_tag, handle_client_exception, prompt, prompt_iterable
+from pcluster.utils import get_supported_os, get_supported_schedulers
+
+from future import standard_library  # isort:skip
```

Comment:
I have no idea, removed.

Reply:
In case we need aliased imports (https://python-future.org/standard_library_imports.html#aliased-imports) we will need to prevent the formatting only of the following lines as it was done initially:

```
# fmt: off
from __future__ import absolute_import, print_function  # isort:skip
from future import standard_library  # isort:skip
standard_library.install_aliases()
# fmt: on
```

whether the aliased imports are still needed I do not know but probably better to leave it there until we have tests to cover compatibility with all python versions.